content/home/marketing/component-based.md

```diff
@@ -1,8 +1,8 @@
 ---
-title: Component-Based
+title: Komponentbaserat
 order: 1
 ---
 
-Build encapsulated components that manage their own state, then compose them to make complex UIs.
+Bygg inkapslade komponenter som hantera sitt egna state, komponera sen ihop dessa för att skapa komplexa användargränssnitt.
 
-Since component logic is written in JavaScript instead of templates, you can easily pass rich data through your app and keep state out of the DOM.
+Eftersom logiken i komponenterna är skriven i JavaScript istället för mallar, så kan du enkelt skicka runt komplex data genom din applikation och hålla state utanför DOM.
```

Comment:
I feel templates in this regard is very close-knitted to this area of concern, so translating it to 'mallar' does not really sound right. Maybe keeping at as 'templates' is the most idiomatic way here.

Again, what to you think?

Reply:
Yeah, this was a hard one. I had trouble mapping the original English text to how I explain this in my React courses to my students. Disregarding the original English text I would write something like "Eftersom logiken i komponenterna är skriven i ren JavaScript så har du tillgång till all funktionalitet som finns i JavaScript. Detta gör det enkelt att skicka runt komplex data genom din applikation och hålla state utanför DOM."